core/src/components/card/card.ionic.scss

```diff
@@ -0,0 +1,35 @@
+@use "../../foundations/ionic.vars.scss" as tokens;
+@import "./card";
```

Comment:
We should not import the partial from ios/md card

Reply:
Done